<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientManager</class>
 <widget class="QWidget" name="ClientManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>ClientDB</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTableView" name="tableView">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Inputpage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>299</width>
         <height>386</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Input</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>ClientID</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>ClientName</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>ClientPhone</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>ClientEmail</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="CIDLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="CPhoneLineEdit">
            <property name="inputMask">
             <string>999-0000-9999</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="CEmailLineEdit"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="CNameLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>아이디를 인풋 할 시 하단의 Recent를 클릭후
인풋 데이터를 해줘야 데이터가 입력된다</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>323</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLineEdit" name="RemoveLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="removeButton">
            <property name="text">
             <string>Recent</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="InputButton">
            <property name="text">
             <string>Input</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="CancelButton">
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ModifyButton">
            <property name="text">
             <string>Modify</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="SearchBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>299</width>
         <height>386</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTreeWidget" name="ClientSearchTree">
          <column>
           <property name="text">
            <string>ClientID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ClientName</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ClientPhone</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ClientEmail</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QComboBox" name="SearchComboBox">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>ClientID</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ClientName</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ClientPhone</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ClientEmail</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="SearchLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="SearchButton">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>CIDLineEdit</tabstop>
  <tabstop>CNameLineEdit</tabstop>
  <tabstop>CPhoneLineEdit</tabstop>
  <tabstop>CEmailLineEdit</tabstop>
  <tabstop>RemoveLineEdit</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>InputButton</tabstop>
  <tabstop>CancelButton</tabstop>
  <tabstop>ModifyButton</tabstop>
  <tabstop>ClientSearchTree</tabstop>
  <tabstop>SearchComboBox</tabstop>
  <tabstop>SearchLineEdit</tabstop>
  <tabstop>SearchButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
